git_machete/cmd.py

```diff
@@ -3469,7 +3468,15 @@ def short_usage() -> None:
 
 
 def version() -> None:
-    print(f"git-machete version {__version__}")
+    try:
+        site_packages_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), '..')
+        info_folder = [f for f in os.listdir(site_packages_path) if f.startswith('git_machete-')][0]
+    except IndexError:
+        # If not found try to look one folder deeper
```

Comment:
Btw `deeper` or rather the opposite, `shallower`? Coz you're adding an extra `..` here